src/core/core.js

```diff
@@ -136,8 +136,8 @@ export default class Core {
    */
   init (config) {
     const params = {
-      token: this._token,
-      apiKey: this._apiKey,
+      ...(this._token && { token: this._token }),
```

Comment:
I don't remember, do we need the parenthesis here? Otherwise, LGTM

Reply:
it looks like you technically don't but I slightly prefer having it

Reply:
it's not needed, but I think it's easier to see that it's spreading out the object in the second condition instead of the first field